static_src/actions/notification_actions.js

```diff
@@ -0,0 +1,29 @@
+
+/*
```

Comment:
@msecret Sounds good.

I initially did a local action/userStore based notification. I hit a problem because of the discard notification dispatch event being called during another dispatch event. Specifically, when I had a close button on the notification, the component was choking on render.

I looked into a solution, and all I saw were adding timeout related solutions, so I thought it would be better to have an external store manage the potential notifications associated to cards.